rdr_service/resource/schemas/genomics.py

```diff
@@ -0,0 +1,222 @@
+    # validation_flags is an array of GenomicValidationFlag Enum values.
+    validation_flags = fields.String(validate=validate.Length(max=80))
+    validated_time = fields.DateTime()
+    sample_id = fields.String(validate=validate.Length(max=80))
+    sample_type = fields.String(validate=validate.Length(max=50))
+    reconcile_cvl_job_run_id = fields.Int32()
+    sequencing_file_name = fields.String(validate=validate.Length(max=128))
+    reconcile_gc_manifest_job_run_id = fields.Int32()
+    reconcile_metrics_bb_manifest_job_run_id = fields.Int32()
+    reconcile_metrics_sequencing_job_run_id = fields.Int32()
+    ai_an = fields.String(validate=validate.Length(max=2))
+    gc_manifest_box_plate_id = fields.String(validate=validate.Length(max=50))
+    gc_manifest_box_storage_unit_id = fields.String(validate=validate.Length(max=50))
+    gc_manifest_contact = fields.String(validate=validate.Length(max=50))
+    gc_manifest_email = fields.String(validate=validate.Length(max=50))
+    gc_manifest_failure_description = fields.String(validate=validate.Length(max=128))
+    gc_manifest_failure_mode = fields.String(validate=validate.Length(max=128))
+    gc_manifest_matrix_id = fields.String(validate=validate.Length(max=20))
+    gc_manifest_parent_sample_id = fields.String(validate=validate.Length(max=20))
+    gc_manifest_quantity_ul = fields.Int32()
+    gc_manifest_sample_source = fields.String(validate=validate.Length(max=20))
+    gc_manifest_study = fields.String(validate=validate.Length(max=50))
+    gc_manifest_study_pi = fields.String(validate=validate.Length(max=50))
+    gc_manifest_test_name = fields.String(validate=validate.Length(max=50))
+    gc_manifest_total_concentration_ng_per_ul = fields.Int32()
+    gc_manifest_total_dna_ng = fields.Int32()
+    gc_manifest_tracking_number = fields.String(validate=validate.Length(max=50))
+    gc_manifest_treatments = fields.String(validate=validate.Length(max=20))
+    gc_manifest_visit_description = fields.String(validate=validate.Length(max=128))
+    gc_manifest_well_position = fields.String(validate=validate.Length(max=10))
+    gem_a1_manifest_job_run_id = fields.Int32()
+    gem_a2_manifest_job_run_id = fields.Int32()
+    gem_pass = fields.String(validate=validate.Length(max=10))
+    gem_a3_manifest_job_run_id = fields.Int32()
+    cvl_aw1c_manifest_job_run_id = fields.Int32()
+    cvl_aw1cf_manifest_job_run_id = fields.Int32()
+    cvl_w1_manifest_job_run_id = fields.Int32()
+    cvl_w2_manifest_job_run_id = fields.Int32()
+    cvl_w3_manifest_job_run_id = fields.Int32()
+    cvl_w4_manifest_job_run_id = fields.Int32()
+    cvl_w4f_manifest_job_run_id = fields.Int32()
+    genomic_workflow_state = fields.EnumString(enum=GenomicWorkflowState)
+    genomic_workflow_state_id = fields.EnumInteger(enum=GenomicWorkflowState)
+
+    genomic_workflow_state_history = fields.JSON()
+    collection_tube_id = fields.String(validate=validate.Length(max=80))
```

Comment:
This field's datatype was changed to an integer in a recent PR: https://github.com/all-of-us/raw-data-repository/pull/1946